README.md

```diff
@@ -32,13 +49,234 @@ import (
 )
 
 func main() {
-    client := imgix.NewClient("mycompany.imgix.net")
+    ub := NewURLBuilder("demo.imgix.net")
+    ixUrl := ub.CreateURL("path/to/image.jpg", url.Values{})
+    // ixUrl == "https://demo.imgix.net/path/to/image.jpg"
+}
+```
+
+```go
+ub := NewURLBuilder("demo.imgix.net")
+ub.CreateURL("path/to/image.jpg", url.Values{"h": []string{"100"}, "w": []string{"100"}})
+// "https://demo.imgix.net/path/to/image.jpg?h=100&w=100"
+```
+
+_HTTPS_ support is enabled by default. _HTTP_ can be toggled on by setting `use_https` to `False`:
+
+```go
+ub := NewURLBuilder("demo.imgix.net")
+ub.SetUseHTTPS(false)
+ub.CreateURL("path/to/image.jpg", url.Values{})
+// "http://demo.imgix.net/path/to/image.jpg"
+```
+
+## Signed URLs
+
+To produce a signed URL, you must enable secure URLs on your source and then provide your signature key to the URL builder.
```

Comment:
I think this should probably be "secret", not "signature"? Not sure though.

```suggestion
To produce a signed URL, you must enable secure URLs on your source and then provide your secret key to the URL builder.
```

Reply:
Yeah, I think we ought to "collect our terms," in the current mix of things we have: sign key, signature, secure, securing, singing, secret, secret key, token,  etc. Let's do some thinking about how to collect terms in a way that complements dashboard. (todo)